openkfw/Oscar
# #43-adding windows scripts to starting package

## Description

In oscar_starting_package scripts are added to start the application on machines with Windows OS similar to existing scripts for macOS. Update that the existing initial data is loaded.

## Motivation and Context



Windows computers that do not have the ability to run bash scripts need batch scripts. Two new scripts have been written for this purpose. In addition, it can happen on Windows computers that an error occurs when starting the application because port 80 is occupied. If this is not an application, but the system itself, an intermediate step must be carried out so that the application can run. The command to release these ports comes from https://superuser.com/questions/352017/pid4-using-port-80. Please check updated Readme.txt. For the initial data, the file ".env" was updated. The value of the field "Country" was updated to "Sample". Now sample data is loaded into the application on start.


## How has this been tested?




Main testing on Windows OS:
1. Download zip oscar_starting_package
2. Extract all files
3. Check for files: start.bat & stop.bat & updated Readme.txt
4. Open terminal
5. Navigate to oscar_starting_package folder ("cd Downloads/oscar_starting_package")
6. (Optional: If port 80 is blocked by the system: "netsh add iplisten ipaddress=0.0.0.0")
7.  Run "start start.bat"
8. Check localhost
9. Run "start stop.bat"

Testing on macOS:
1. Same steps as above till step 5
2. Open terminal 
3. Run "sh ./start.bat"
4. There will be an error as the @echo command is not found
5. Application should start nevertheless
6. Run "sh ./stop.bat"


## Screenshots (if appropriate):

## Types of changes




## Checklist:
